man/bst-show.1

```diff
@@ -45,6 +45,7 @@ Show elements in the pipeline
         %{deps}           A list of all dependencies
         %{build-deps}     A list of build dependencies
         %{runtime-deps}   A list of runtime dependencies
+        %{cas-digest}     The CAS digest
```

Comment:
A bit of bike shedding:
* "The CAS digest" is too short to explain what this does. The CAS digest of what exactly?
* I feel that `cas-digest` is too generic. This is actually the cas digest of the built artifact, we could (maybe not now, but in the future) have cas digests of other things (like the source code, or the buildtree if we have one)